MixItUp.AutoHoster/App.config

```diff
@@ -45,6 +45,58 @@
         <assemblyIdentity name="Microsoft.Owin" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
Sorry, but the .NET Standard 2.0 stuff requires projects that use it to references these. :(